enterprise/migrations/0133_auto_20210608_1931.py

```diff
@@ -0,0 +1,326 @@
+                                            </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </td>
+                    </tr>
+                    <tr>
+                    <td>
+                        <table cellpadding="0" cellspacing="0" width="100%">
+                            <tbody>
+                                <tr>
+                                <td height="20" style="line-height:1px;font-size:1px"></td>
+                                </tr>
+                                <tr>
+                                <td align="left" valign="top"><a href="https://business.edx.org" style="font-family:'Open Sans',Arial,sans-serif;font-size:14px;line-height:17px;text-decoration:none;color:#707070" target="_blank"><span style="color:#00262b">edX for Business</span><span style="color:#707070"> — eLearning Solutions for Your Company</span></a></td>
+                                </tr>
+                                <tr>
+                                <td height="20" style="line-height:1px;font-size:1px"></td>
+                                </tr>
+                                <tr>
+                                <td align="left" style="font-family:'Open Sans',Arial,sans-serif;color:#707070;font-size:14px;line-height:17px" valign="top">© 2021 edX Inc. All rights reserved.</td>
+                                </tr>
+                                <tr>
+                                <td height="20" style="line-height:1px;font-size:1px"></td>
+                                </tr>
+                                <tr>
+                                <td height="20" style="line-height:1px;font-size:1px"></td>
+                                </tr>
+                                <tr>
+                                <td align="left" style="font-family:'Open Sans',Arial,sans-serif;color:#707070;font-size:14px;line-height:17px" valign="top">141 Portland St. 9th Floor, Cambridge, MA 02139</td>
+                                </tr>
+                                <tr>
+                                <td height="30" style="line-height:1px;font-size:1px"></td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </td>
+                    </tr>
+                </tbody>
+            </table>
+        </body>
+        </html>
+        """,
+        subject_line='',
+        enterprise_customer=None,
+        template_type='ADMIN_ENROLL',
```

Comment:
Have you by chance had the ability to actually see the rendered email from this? I'm just curious if this hideous monstrosity actually does save and render properly (I know it's not your doing for why it is hideous, that's just what it has to look like. I'm just wondering if it does render properly to and from the database). If you haven't been able to test, let's just double-check that on stage before we deploy this to prod when we get there. 👍

Reply:
ah yes sorry sharing image in description section

Reply:
I do expect to have to make changes to the template btw so this may not be final, I am certain Jen or Joe are going to have feedback. I also need to fix that My Dashboard style but I wanted to see if I can do that from django admin once this is live...